recipes/pyarrow/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.1.pre" %}
+{% set git_sha = "2d8e82056afdcf125e6e512f96007389ce79c1c7" %}
+
+package:
+  name: pyarrow
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/apache/arrow.git
```

Comment:
Does the feedback from @ocefpaf on arrow-cpp [here](https://github.com/conda-forge/staged-recipes/pull/1729#pullrequestreview-3494413) also apply?